test/fartgate/core_test.clj

```diff
@@ -1,5 +1,7 @@
 (ns fartgate.core-test
   (:require [clojure.test :as t]
-            [fartgate.core :as f]))
+            [fartgate.core :as f]
+            [cheshire.core :as json]))
+
+(t/deftest default (t/is (= (f/-handleRequest nil (clojure.java.io/reader "dev-resources/test.json") nil nil) {:container 137})))
```

Comment:
1. require clojure.java.io :as io so you don't spell it out fully (and explicitly require it instead of implicitly assuming it's around)
2. use `(io/resource "test.json")` so your test doesn't depend on `dev-resources` (you'll get away with it here because you're likely only going to run the tests in this specific setup -- but it's still bad practice)